src/LookingGlassControls.ts

```diff
@@ -194,6 +193,30 @@ export function initLookingGlassControlGUI() {
 				stringify: (v) => (v === 0 ? "swizzled" : v === 1 ? "center" : v === 2 ? "quilt" : "?"),
 			}
 		)
+
+		addControl(
+			"filterMode",
+			{ type: "range", min: 0, max: 3, step: 1 },
+			{
+				label: "view filtering mode",
+				title: "controls the method used for view blending",
+				fixRange: (v) => Math.max(0, Math.min(v, 2)),
+				stringify: (v) => (v === 0 ? "old, studio style" : v === 1 ? "2 view" : v === 2 ? "gaussian" : v === 3 ? "10 view gaussian": "?"),
+			}
+		)
+
+		addControl(
+			"gaussianSigma",
+			{type: "range", min: -1, max: 1, step: 0.02},
```

Comment:
Might want the step here to be 0.01, given that, IIRC, the default should be 0.01.

Reply:
good catch here, the default itself should be 0.02 from what I discussed with Alec